src/components/ImportPage.tsx

```diff
@@ -23,14 +23,14 @@ const ImportPage: React.FunctionComponent<PipelineWizardPageProps> = ({
     </Helmet>
     <QueryClientProvider client={queryClient}>
       <NamespaceContext.Provider value={namespace}>
-        <Page>
+        <>
```

Comment:
For my own edification, what is happening here with this change? a no-op?

Reply:
Ah, because we're embedding this bit of UI within another UI that's already accounting for the page's primary `main` region - the markup generated by the Page component here was creating navigation issues for screen reader users by redundantly adding another main region to the page (also other duplicate/unnecessary markup structures)